app/components/TaskSheet/TaskList.tsx

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import TaskItem from "./TaskItem";
+import { Task } from "@/app/api/interfaces/interfaces";
+
+interface TaskListProps {
+  tasks: Task[];
+  onTaskCompletion: (taskName: string) => void;
```

Comment:
taskName not being used is causing the workflows to fail

Reply:
I have been spending this afternoon looking into this but cannot figure out why the error is happening. Would love some help.